bc_obps/registration/utils.py

```diff
@@ -242,3 +242,39 @@ def custom_reverse_lazy(view_name, *args, **kwargs) -> str:
 def set_verification_columns(record, user_guid):
     record.verified_at = datetime.now(pytz.utc)
     record.verified_by_id = user_guid
+
+
+def files_have_same_hash(file1, file2) -> bool:
+    """
+    Compare the hash of two files to determine if they are the same.
+    this might miss formatting changes.
```

Comment:
I believe that it should be alright since users will download the template, fill it out, and then upload it, so they are unlikely to modify the layout of the PDF unless they intend to upload something other than a statutory declaration. It's important to note that comparing layout changes would be a time-consuming and memory-intensive task, so I would not recommend doing so unless it's absolutely necessary. That being said, I'm tagging @nanyangpro and @andrea-williams to get their input on this matter.

Reply:
it's very unlikely that a format change could happen that wouldn't result in a change to the hash value (only circumstance where it wouldn't change would be if a hash collision occurred). The hashing also includes the file metadata. I think it's safe to assume that any changes to the file will be detected with this strategy.